project-base/build.xml

```diff
@@ -10,7 +10,7 @@
     <property name="path.framework.assets" value="${path.root}/node_modules/@shopsys/framework"/>
     <property name="path.frontend-api" value="${path.vendor}/shopsys/frontend-api"/>
 
-    <property name="phpstan.level" value="4"/>
+    <property name="phpstan.level" value="5"/>
```

Comment:
I do not think it is good idea to enforce phpstan level 5 in projects

Reply:
It is not enforcing as it is an optional update. Only new projects will start with this level so it is OK to me.

Reply:
you should then say in Upgrade Notes that this is up to developer to decide if he wants to increase phpstan level